package/Dockerfile.lite

```diff
@@ -84,10 +87,7 @@ COPY package/etc/conf.d/plugin /etc/syslog-ng/conf.d/plugin
 COPY package/etc/conf.d/sc4slib /etc/syslog-ng/conf.d/sc4slib
 COPY package/etc/conf.d/sources /etc/syslog-ng/conf.d/sources
 
-COPY package/lite/etc/conf.d/conflib/raw/app-raw-bsd_nopri.conf /etc/syslog-ng/conf.d/conflib/raw/app-raw-bsd_nopri.conf
-COPY package/lite/etc/conf.d/log_paths/2/lp_dest_alts_global/plugin.jinja /etc/syslog-ng/conf.d/log_paths/2/lp_dest_alts_global/plugin.jinja
-COPY package/lite/etc/conf.d/plugin/app_parser_topics.conf /etc/syslog-ng/conf.d/plugin/app_parser_topics.conf
-COPY package/lite/etc/conf.d/sources/source_syslog/plugin.jinja /etc/syslog-ng/conf.d/sources/source_syslog/plugin.jinja
+COPY package/lite/etc/conf.d/conflib /etc/syslog-ng/conf.d/conflib
```

Comment:
after this changes sc4s lite will have self tiny conflib with only generic configs (no vendor-specific). Currently SC4S lite more similar on syslog-ng, because it's working with 3-5 topics